tests/kolainst/kolainst-build.sh

```diff
@@ -53,5 +53,8 @@ build_rpm testdaemon \
                   useradd -r testdaemon-user -g testdaemon-group -s /sbin/nologin" \
              files "/usr/bin/%{name}
                     /var/lib/%{name}"
+# Will be useful for testing cancellation
+build_rpm testpkg-post-infinite-loop \
```

Comment:
This doesn't seem used in any new test.

Reply:
Yeah, I was just using it locally to test.